web_src/css/base.css

```diff
@@ -2367,3 +2367,11 @@ table th[data-sortt-desc] .svg {
   width: 15px;
   height: 15px;
 }
+
+.ui.button.flex-text-wrap,
+.flex-text-wrap {
+  display: inline-flex;
```

Comment:
`inline-flex` may introduce unexpected inline rendering, so I recommend `flex` (block mode). This is already one issue why you can't make such generic classes, sometimes one wants `inline`, sometimes one wants `block`.

Reply:
By default, most elements like "ui button" are "inline-block". So, "inline-flex" is good enough to replace "inline-block"

I have considered about this problem, IMO if we need to use "display: flex", we can do `flex-text-wrap gt-df`, then everything is still good.

Reply:
Or maybe we can use separate CSS names, eg: `flex-inline-wrap` vs `flex-block-wrap`

Reply:
`flex-text-block` and `flex-text-inline` would be my preference. Most if not all text should wrap, so it doesn't need special attention in the name.